src/main/webapp/js/finscholar/navbar/navbar.soy

```diff
@@ -13,21 +13,22 @@
 // limitations under the License.
 {namespace finscholar.navbar.templates}
 {template .navbar}
-  <nav id="navbar">
-    <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect 
-        mdl-button--accent" index="0" 
-        jsaction="navbar.clickAction;dblclick:navbar.doubleClickAction">
-      Home
-    </button>
-    <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect
-        mdl-button--accent" index="1" 
-        jsaction="navbar.clickAction;dblclick:navbar.doubleClickAction">
-      College List
-    </button>
-    <button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect 
-        mdl-button--accent" index="2" 
-        jsaction="navbar.clickAction;dblclick:navbar.doubleClickAction">
-      Scholarship List
-    </button>
-  </nav>
+  <header class="mdl-layout__header">
```

Comment:
Nice. I thought about using this, but I wasn't sure if it would break the existing layout.